src/map/utils/synthutils.cpp

```diff
@@ -340,25 +340,25 @@ namespace synthutils
             switch (finalHQTier)
             {
                 case 4: // 1 in 2
-                    chanceHQ = 50.0;
+                    chanceHQ = settings::get<float>("map.SYNTHESIS_TIER_4_HQ_CHANCE");
                     break;
                 case 3: // 1 in 4
-                    chanceHQ = 25.0;
+                    chanceHQ = settings::get<float>("map.SYNTHESIS_TIER_3_HQ_CHANCE");
                     break;
                 case 2: // 1 in 20
-                    chanceHQ = 5.0;
+                    chanceHQ = settings::get<float>("map.SYNTHESIS_TIER_2_HQ_CHANCE");
```

Comment:
This might be a little out of scope for this PR but I will leave this here anyway so its at least somewhere. I believe this crafting rate should be 6.25 (1/16) instead of (1/20). Most likely the same for T0 is 1/64 instead of 1/100 but thats been very difficult to prove. There are very large sample sizes from Byrth here: https://www.bluegartr.com/threads/120352-CraftyMath

The 5% came from the JP wiki which is only based off 1000 crafts in each tier which has an HQ T1 rate of 5.82%.  https://wiki.ffo.jp/html/843.html

SE said the HQ rates were increased and people always guessed T1 was changed from 1/16 to 1/10 but from all the sample sizes it was never proven as far as I know and SE never came out to say what the rates actually were. 
![image](https://github.com/LandSandBoat/server/assets/105882754/e255eaef-ac28-4787-8289-f9ab02108f79)

Reply:
Remember that when in doubt, we always go with a "in players detriment" choice.

Boosting a rate is ok for a server OP.
Having to nerf it, however, causes drama and the damage has already be done.

Reply:
You don't have to tell me that ;) haha

Reply:
Well, it was for everyone, not specifically for you.

This kind of stuff has to be said on occasions. New ppl come and dont now the reasoning behind it.

Reply:
I can certainly tweak this to 6.25, would y'all prefer a separate PR for that? Or are we fine with keeping it at 5.0 for now?

Reply:
For the time being, all values should be kept as they are.

Reply:
Reverted.